fixture-monkey-tests/kotlin-tests/src/test/kotlin/com/navercorp/fixturemonkey/tests/kotlin/SimpleValueJqwikPluginTest.kt

```diff
@@ -157,4 +157,16 @@ class SimpleValueJqwikPluginTest {
         then(setObject).isNotNull
         then(setObject.integers).hasSize(3)
     }
+
+    @RepeatedTest(TEST_COUNT)
+    fun sampleByte() {
```

Comment:
It would be appreciated if you could add new test cases to set the `minNumberValue` option to `-129` and the `maxNumberValue` option to `129`.

Reply:
Thank you for your feedback.
I have committed 3 fixes.

- https://github.com/naver/fixture-monkey/pull/988/commits/53b188f5c97c2e4f12eaeb9d0fe40bff4b206d8b